From 5ea351cd6bc662ebcee2e322f89b55fba2d0b452 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 14 Mar 2025 20:47:13 -0700
Subject: [PATCH 01/21] use local bazel to workaround bazel paralle issue

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
Signed-off-by: Julien STEPHAN <jstephan@baylibre.com>

Rebase to 2.19
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.py b/configure.py
index ec04fcfdd0c..5d39e4163a4 100644
--- a/configure.py
+++ b/configure.py
@@ -420,7 +420,7 @@ def retrieve_bazel_version():
   Returns:
     The bazel version detected.
   """
-  bazel_executable = shutil.which('bazel')
+  bazel_executable = "%%WORKDIR%%/bazel/bazel"
   if bazel_executable is None:
     bazel_executable = shutil.which('bazelisk')
     if bazel_executable is None:
-- 
2.25.1

