From 6bb9c02aae20996a6e4edd8fe743a90ce8d677aa Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sun, 18 Sep 2022 23:52:34 -0700
Subject: [PATCH] drop bazel_toolchains_repositories to support offline build

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase to 2.10.0
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tensorflow/workspace0.bzl | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tensorflow/workspace0.bzl b/tensorflow/workspace0.bzl
index c3c081b19ef..445de71bb13 100644
--- a/tensorflow/workspace0.bzl
+++ b/tensorflow/workspace0.bzl
@@ -2,7 +2,6 @@
 
 load("//third_party/googleapis:repository_rules.bzl", "config_googleapis")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("@bazel_toolchains//repositories:repositories.bzl", bazel_toolchains_repositories = "repositories")
 load("@build_bazel_rules_apple//apple:repositories.bzl", "apple_rules_dependencies")
 load("@build_bazel_rules_swift//swift:repositories.bzl", "swift_rules_dependencies")
 load("@build_bazel_apple_support//lib:repositories.bzl", "apple_support_dependencies")
@@ -104,8 +103,6 @@ def workspace():
         ],
     )
 
-    bazel_toolchains_repositories()
-
     # Apple rules for Bazel. https://github.com/bazelbuild/rules_apple.
     # Note: We add this to fix Kokoro builds.
     # The rules below call into `rules_proto` but the hash has changed and
-- 
2.37.1

