From 96bee927e77c384a22e647f15c24209b9a30e60d Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 17 Aug 2021 14:58:33 +0800
Subject: [PATCH] drop bazel_toolchains_repositories to support offline build

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tensorflow/workspace0.bzl | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tensorflow/workspace0.bzl b/tensorflow/workspace0.bzl
index 852d094922d..3adad3fb8c4 100644
--- a/tensorflow/workspace0.bzl
+++ b/tensorflow/workspace0.bzl
@@ -2,7 +2,6 @@
 
 load("//third_party/googleapis:repository_rules.bzl", "config_googleapis")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("@bazel_toolchains//repositories:repositories.bzl", bazel_toolchains_repositories = "repositories")
 load("@build_bazel_rules_swift//swift:repositories.bzl", "swift_rules_dependencies")
 load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
 load("@local_config_android//:android.bzl", "android_workspace")
@@ -105,8 +104,6 @@ def workspace():
 
     rules_cc_toolchains()
 
-    bazel_toolchains_repositories()
-
     # Use `swift_rules_dependencies` to fetch the toolchains. With the
     # `git_repository` rules above, the following call will skip redefining them.
     swift_rules_dependencies()
-- 
2.27.0

