From 104224b8b30f74dabed893aa057f400ad7cd2695 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 16 May 2022 00:10:55 -0700
Subject: [PATCH] drop v2 config

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 .bazelrc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/.bazelrc b/.bazelrc
index b6a72c1e5d3..a71ade6950f 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -138,9 +138,6 @@ build --define=with_xla_support=true
 # TODO(mihaimaruseac): Document this option or remove if no longer needed
 build --config=short_logs
 
-# TODO(mihaimaruseac): Document this option or remove if no longer needed
-build --config=v2
-
 # Disable AWS/HDFS support by default
 build --define=no_aws_support=true
 build --define=no_hdfs_support=true
-- 
2.36.0

