From bbdf1bbca17f8bce98e32b25aa9bfadc8f465f24 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sun, 15 May 2022 23:59:17 -0700
Subject: [PATCH] use local bazel to workaround bazel paralle issue

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
Signed-off-by: Julien STEPHAN <jstephan@baylibre.com>
---
 configure.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.py b/configure.py
index dae3dce89d3..979f00f3633 100644
--- a/configure.py
+++ b/configure.py
@@ -475,7 +475,7 @@ def check_bazel_version(min_version, max_version):
   Returns:
     The bazel version detected.
   """
-  bazel_executable = which('bazel')
+  bazel_executable = "%%WORKDIR%%/bazel/bazel"
   if bazel_executable is None:
     bazel_executable = which('bazelisk')
     if bazel_executable is None:
-- 
2.36.0

