From 856e20e86e771224963d5791d57c8f6e802b1450 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sun, 15 May 2022 23:59:17 -0700
Subject: [PATCH] use local bazel to workaround bazel paralle issue

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
Signed-off-by: Julien STEPHAN <jstephan@baylibre.com>
---
 configure.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.py b/configure.py
index bf338bdda22..034e7c7c67f 100644
--- a/configure.py
+++ b/configure.py
@@ -420,7 +420,7 @@ def retrieve_bazel_version():
   Returns:
     The bazel version detected.
   """
-  bazel_executable = which('bazel')
+  bazel_executable = "%%WORKDIR%%/bazel/bazel"
   if bazel_executable is None:
     bazel_executable = which('bazelisk')
     if bazel_executable is None:
