From c4cf6b4eb20900956601b4bcdd237a9b33c89fd6 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 22 Aug 2024 01:42:24 -0700
Subject: [PATCH 03/21] drop bazel_toolchains_repositories to support offline
 build

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase to 2.17.0
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tensorflow/workspace0.bzl | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tensorflow/workspace0.bzl b/tensorflow/workspace0.bzl
index 0fdb0342fae..d617f7d02d2 100644
--- a/tensorflow/workspace0.bzl
+++ b/tensorflow/workspace0.bzl
@@ -1,6 +1,5 @@
 """TensorFlow workspace initialization. Consult the WORKSPACE on how to use it."""
 
-load("@bazel_toolchains//repositories:repositories.bzl", bazel_toolchains_repositories = "repositories")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@build_bazel_apple_support//lib:repositories.bzl", "apple_support_dependencies")
 load("@build_bazel_rules_apple//apple:repositories.bzl", "apple_rules_dependencies")
@@ -105,8 +104,6 @@ def workspace():
         ],
     )
 
-    bazel_toolchains_repositories()
-
     # Apple rules for Bazel. https://github.com/bazelbuild/rules_apple.
     # Note: We add this to fix Kokoro builds.
     # The rules below call into `rules_proto` but the hash has changed and
-- 
2.25.1

