From c8b9cc1bdfcfd9aedd4f79a50cc3c3e3e6a289ee Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 14 Mar 2025 20:52:17 -0700
Subject: [PATCH 04/21] drop v2 config

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 .bazelrc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/.bazelrc b/.bazelrc
index b9f83146895..be80729ac91 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -111,9 +111,6 @@ build --define=with_xla_support=true
 # TODO(mihaimaruseac): Document this option or remove if no longer needed
 build --config=short_logs
 
-# TODO(mihaimaruseac): Document this option or remove if no longer needed
-build --config=v2
-
 # TF now has `cc_shared_library` targets, so it needs the experimental flag
 # TODO(rostam): Remove when `cc_shared_library` is enabled by default
 build --experimental_cc_shared_library
-- 
2.25.1

